Movie_Ticket_Booking/src/main/java/com/org/Movie_Ticket_Booking/controller/cinemaManager/CinemaController.java

```diff
@@ -0,0 +1,67 @@
+package com.org.Movie_Ticket_Booking.controller.cinemaManager;
```

Comment:
```suggestion
package com.org.Movie_Ticket_Booking.controller.manager;
```
chị nghĩ nếu là phân role thì nó là manager chứ nhỉ